gprofiler/main.py

```diff
@@ -554,7 +587,14 @@ def parse_cmd_args() -> configargparse.Namespace:
     for subparser in [parser, upload_file]:
         connectivity = subparser.add_argument_group("connectivity")
         connectivity.add_argument(
-            "--server-host", default=GRANULATE_SERVER_HOST, help="Server host (default: %(default)s)"
+            "--server-host",
+            default=DEFAULT_PROFILER_SERVER_ADDRESS,
+            help="URL for uploading profiles (default: %(default)s)",
```

Comment:
Technically it's not a URL but the server address, let's use "Server host" or "Server address"? Same for `--api-server`